invenio_app_ils/ui/backoffice/src/pages/ItemDetails/ItemDetailsContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { ItemDetails } from './components/ItemDetails/ItemDetails';
+
+export default class ItemDetailsContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.fetchItemDetails = this.props.fetchItemDetails;
+  }
+
+  componentDidMount() {
+    this.unlisten = this.props.history.listen((location, action) => {
+      if (location.state && location.state.itemId) {
+        this.fetchItemDetails(location.state.itemId);
+      }
+    });
+    this.fetchItemDetails(this.props.match.params.itemId);
+  }
+
+  componentWillUnmount() {
+    this.unlisten();
+  }
+
+  render() {
+    return <ItemDetails {...this.props} />;
```

Comment:
IMHO, if you use this "container" logic, then you should pass to the "display" container only the data itself and not all `this.props`, no?
So, `<ItemDetails isLoading={} results={} />`